sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/EncryptParameters.java

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.cryptography;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.security.keyvault.keys.cryptography.models.EncryptionAlgorithm;
+
+import java.util.Objects;
+
+/**
+ * A class containing various configuration parameters that can be applied when performing encryption operations.
+ */
+public class EncryptParameters {
+    /**
+     * The algorithm to be used for encryption.
+     */
+    private final EncryptionAlgorithm algorithm;
+
+    /**
+     * The content to be encrypted.
+     */
+    private final byte[] plainText;
+
+    /**
+     * Initialization vector to be used in the encryption operation using a symmetric algorithm.
+     */
+    private final byte[] iv;
```

Comment:
Just so I make sure it's called out, where is this generated if not specified only for AES-CBC(PAD)? Note that it *must* be a cryptographic RNG.

Reply:
I found it in the `SymmetricKeyCryptographicClient`, but still want to keep this comment open just so that you make sure it's correctly generated only for AES-CBC(PAD) and never AES-GCM.